sample.env

```
@@ -94,7 +94,7 @@ REPOSITORY=islandora
 
 # The version of the isle-buildkit images, non isle-buildkit images have
 # their versions specified explicitly in their respective docker-compose files.
-TAG=3.0.0
+TAG=3.1.0
```

```suggestion
TAG=3.1.2
```

[3.1.0](https://github.com/Islandora-Devops/isle-buildkit/releases/tag/3.1.0) upgrades solr server from 8.x to 9.x... If we're bumping the isle-buildkit tag for Cantaloupe support, I suggest we pin to [3.1.2](https://github.com/Islandora-Devops/isle-buildkit/releases/tag/3.1.2) which is the first point release for cantaloupe in several years and resolves several security issues.